Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -208,8 +203,12 @@
           },
           {
             "target": "versions",
-            "dataSourceName": "Builds",
-            "resultTemplate": "{ Value : \"{{id}}\", DisplayValue : \"{{{number}}}\" }"
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any,count:2000,start:0{{skip}}",
+            "resultSelector": "jsonpath:$.build[?(@.status=='SUCCESS')]",
+            "resultTemplate": "{ Value : \"{{id}}\", DisplayValue : \"{{{number}}}\" }",
+            "callbackContextTemplate": "{\"skip\": \"{{add skip 2000}}\"}",
+            "callbackRequiredTemplate": "{{ result.nextHref }}",
```

Comment:
What does result.nextHref mean here?
Can you share your response structure here for me to validate?

Reply:
Paginated results in the TeamCity API includes a url for the next and previous pages, but only if there is one:

```
{
  "count": 1000, 
  "href": "/httpAuth/app/rest/buildTypes/MyProject_Default/builds/?locator=branch:default:any,count:1000,start:01",
  "nextHref": "/httpAuth/app/rest/buildTypes/MyProject_Default/builds/?locator=branch:(default:any),count:1000,start:1001", 
  "prevHref": "/httpAuth/app/rest/buildTypes/MyProject_Default/builds/?locator=branch:(default:any),count:1,start:0", 
  "build": [
  ... actual build details go here
  ]
}
```

So the intention was just to check that this property is defined and non-empty.